docs/specifications/tests/DNSSEC-TP/dnssec15.md

```diff
@@ -0,0 +1,173 @@
+# DNSSEC15: Existence of CDS and CDNSKEY
+
+## Test case identifier
+**DNSSEC15**
+
+## Objective
+
+CDS and CDNSKEY record types are defined in [RFC 7344] and [RFC 8078].
+Both record types are optional in a zone. The objective of this test
+case is to verify that they are correctly set-up, if included in the
+zone.
+
+If a CDS record is included in the zone, the corresponding CDNSKEY
+record should also be included ([RFC 7344][RFC 7344, section 4],
+section 4).
+
+The CNS and CDNSKEY RRsets should be consistent between all name
+servers for the zone in question.
+
+If there are both CDS RRs and CDNSKEY RRs in the zone they must match in 
+content ([RFC 7344][RFC 7344, section 4], section 4). It means that both
+must be derived from the same DNSKEY or both being "delete" CDS and
+CDNSKEY.
+
+## Scope
+
+It is assumed that *Child Zone* has been tested by [Basic04]. This test
+case will just ignore non-responsive name servers or name servers not
+giving a correct DNS response for an authoritative name server.
+
+## Inputs
+
+* "Child Zone" - The domain name to be tested.
+
+## Summary
+
+Message Tag outputted     | [Default level] | Description of when message tag is outputted
```

Comment:
This link label doesn't match any link reference definition.

Reply:
Will fix.